CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,142 @@
+# Dataverse&reg; Code of Conduct
```

Comment:
Shouldn't the Code of Conduct be the same as https://github.com/IQSS/dataverse/blob/develop/CODE_OF_CONDUCT.md ? (It's what we link to from https://dataverse.org as well.)

Reply:
Done! I replaced it with the CODE_OF_CONDUCT from the main Dataverse repo